server/interfaces/interfaces.go

```diff
@@ -1656,7 +1656,8 @@ type CPULeaser interface {
 }
 
 const (
-	OCIImageInstanceNamePrefix = "_bb_ociregistry_"
+	OCIImageInstanceNamePrefix           = "_bb_ociregistry_"
```

Comment:
The motivation for making this a prefix was so that it was possible to store various kinds of registry artifacts under the prefix without having to update this. Could you have the registry code place things under _bb_ociregistry_/foo instead of introducing another prefix?

Reply:
Fixed.